stackdriver-debugger/src/com/google/cloud/tools/intellij/stackdriver/debugger/CloudDebugSettingsEditor.java

```diff
@@ -41,11 +39,14 @@
 
   @Override
   protected void applyEditorTo(@NotNull CloudDebugRunConfiguration runConfiguration) {
-    // TODO(ivanporty) CloudDebugRunConfiguration uses `cloudProjectName` for project ID.
-    runConfiguration.setCloudProjectName(
+    runConfiguration.setCloudProjectId(
         Optional.ofNullable(settingsPanel.getSelectedCloudProject())
             .map(CloudProject::projectId)
             .orElse(""));
+    runConfiguration.setGoogleUsername(
+        Optional.ofNullable(settingsPanel.getSelectedCloudProject())
+            .map(CloudProject::googleUsername)
+            .orElse(null));
```

Comment:
why null here but empty string for the project ID?

Reply:
Should be null for project ID too, changed.